aggregator_api/src/lib.rs

```diff
@@ -441,10 +416,10 @@ mod models {
         pub(crate) time_precision: Duration,
         /// HPKE configuration for the collector.
         pub(crate) collector_hpke_config: HpkeConfig,
-        /// If this aggregator is the leader, this is the bearer token to use to authenticate
-        /// requests to the helper, as Base64 encoded bytes. If this aggregator is the helper, the
-        /// value is `None`.
-        pub(crate) aggregator_auth_token: Option<String>,
+        /// If this aggregator is the leader, this is the token to use to authenticate requests to
+        /// the helper, as Base64 encoded bytes. If this aggregator is the helper, the value is
+        /// `None`.
```

Comment:
nit: the token is now a composite value

```suggestion
        /// If this aggregator is the leader, this is the token to use to authenticate requests to
        /// the helper. If this aggregator is the helper, the value is `None`.
```